packages/frontend/editor-ui/src/features/ai/chatHub/composables/availableTools.ts

```diff
@@ -0,0 +1,104 @@
+import { type ChatHubAgentTool } from '@n8n/api-types';
+import { type INode, JINA_AI_TOOL_NODE_TYPE, SEAR_XNG_TOOL_NODE_TYPE } from 'n8n-workflow';
+
+export interface ChatHubToolProvider {
+	name: string;
+	description: string;
+	credentialType: string | undefined;
+	tools: Array<{
+		title?: string;
+		node: INode;
+	}>;
+}
+
+export const AVAILABLE_TOOLS: Record<ChatHubAgentTool, ChatHubToolProvider> = {
+	[JINA_AI_TOOL_NODE_TYPE]: {
+		name: 'Jina AI',
+		description: 'Use Jina AI to search the web for relevant information.',
+		credentialType: 'jinaAiApi',
+		tools: [
+			{
+				title: 'Access Web',
+				node: {
+					parameters: {
+						resource: 'reader',
+						operation: 'read',
+						url: "={{ /*n8n-auto-generated-fromAI-override*/ $fromAI('URL', ``, 'string') }}",
+						options: {},
+						requestOptions: {},
+					},
+					type: JINA_AI_TOOL_NODE_TYPE,
+					typeVersion: 1,
+					position: [0, 0],
+					id: '23214834-7382-487b-b71b-e9c339fe44aa',
```

Comment:
These have been granted random UUIDs that should remain stable - we use these to restore selected tools when restoring a session from backend data.

We only store `INodes` on the column so I couldn't add the ID on the higher level `ChatHubToolProvider` object & just node version & type isn't enough as here for instance we're providing three variants of the same tool node..